src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/SlimefunTabCompleter.java

```diff
@@ -25,13 +28,15 @@ public SlimefunTabCompleter(SlimefunCommand command) {
     }
 
     @Override
-    public List<String> onTabComplete(CommandSender sender, Command cmd, String label, String[] args) {
+    public List<String> onTabComplete(@Nonnull CommandSender sender, @Nonnull Command cmd, @Nonnull String label, String[] args) {
         if (args.length == 1) {
             return createReturnList(command.getTabArguments(), args[0]);
         }
         else if (args.length == 3) {
             if (args[0].equalsIgnoreCase("give")) {
-                return createReturnList(SlimefunPlugin.getRegistry().getEnabledSlimefunItemIds(), args[2]);
+                return createReturnList(SlimefunPlugin.getRegistry().getEnabledSlimefunItemIds().stream()
+                        .map(String::toLowerCase)
+                        .collect(Collectors.toList()), args[2]);
```

Comment:
This right there is just an absolute nightmare performance-wise...
And it is completely unnecessary too.

Reply:
That's what fixed the bug... I could just make it try to set everything to lowercase even if it is already lowercase inside the method if that is what you prefer.